<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QMainWindow" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adminstrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="pageContainer">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="adminRightSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>347</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="mainSeparator">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="artistDescriptionLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>The list of the artists created for the whole system:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QListView" name="artistView"/>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QHBoxLayout" name="artistControlLayout">
          <item>
           <spacer name="artistControlLeftSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/images/dialogs/add.png</normaloff>:/images/dialogs/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/images/dialogs/delete.png</normaloff>:/images/dialogs/delete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="artistControlRightSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="adminInfoLayout">
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="adminPhotoLabel">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../resources.qrc">:/images/windows/person.png</pixmap>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLabel" name="adminNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>15</pointsize>
               <strikeout>false</strikeout>
               <kerning>false</kerning>
               <fontweight>ExtraLight</fontweight>
              </font>
             </property>
             <property name="text">
              <string>Adminstrator's Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="changeInfoButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Change Information</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="changeInfoSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="informationPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0" colspan="3">
         <spacer name="checkTopSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2">
         <spacer name="checkRightSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <spacer name="checkLeftSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QGroupBox" name="informationLayout">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="firstNameEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>Example: Amir Mohsen</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="firstNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>First Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="lastNameEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>Example: Ghasemi</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lastNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Last Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="passwordLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="showPasswordButton">
             <property name="toolTip">
              <string>Show Password</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../resources.qrc">
               <normaloff>:/images/windows/show-password.png</normaloff>:/images/windows/show-password.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="passwordEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="userNameEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>Example: amir@1384</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="userNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>User Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="infoSeparator">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QGridLayout" name="descriptionLayout">
          <item row="0" column="0" rowspan="2">
           <widget class="QLabel" name="mainTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <fontweight>Light</fontweight>
             </font>
            </property>
            <property name="text">
             <string>Change the information of the adminstrator</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="subtitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Here you can alter different pieces of information.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <widget class="QPushButton" name="cancelButton">
            <property name="toolTip">
             <string>Go Back</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/images/windows/back.png</normaloff>:/images/windows/back.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="checkButton">
          <property name="toolTip">
           <string>Accept</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/images/windows/check.png</normaloff>:/images/windows/check.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>changeInfoButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>goToInfoChangingMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>rejectChanging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminPanel</receiver>
   <slot>acceptChanging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>711</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>741</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPasswordButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>AdminPanel</receiver>
   <slot>togglePasswordShow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goToInfoChangingMode()</slot>
  <slot>rejectChanging()</slot>
  <slot>acceptChanging()</slot>
  <slot>togglePasswordShow(bool)</slot>
 </slots>
</ui>
